### PR TITLE
Update indexSumarize.py

### PR DESCRIPTION
**Pull request by malayjoshi13**

### CODE REVIEW

File: server/src/indexSumarize.py [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-import ast
+
 # -------------------------------------------------------------------------------------------------------------------------------
-model = AutoModelForSeq2SeqLM.from_pretrained('facebook/bart-large-cnn') # google/pegasus-newsroom or facebook/bart-large-cnn or Yale-LILY/brio-cnndm-cased
-tokenizer = AutoTokenizer.from_pretrained('facebook/bart-large-cnn') 
+# model = AutoModelForSeq2SeqLM.from_pretrained('facebook/bart-large-cnn') # google/pegasus-newsroom or facebook/bart-large-cnn or Yale-LILY/brio-cnndm-cased
+# tokenizer = AutoTokenizer.from_pretrained('facebook/bart-large-cnn') 
+
+model = AutoModelForSeq2SeqLM.from_pretrained('./bart-large-cnn') # google/pegasus-newsroom or facebook/bart-large-cnn or Yale-LILY/brio-cnndm-cased
+tokenizer = AutoTokenizer.from_pretrained('./bart-large-cnn')
```
Comment (khareyash05) [inline server/src/indexSumarize.py:9]: again the model change can cause errors

File: server/src/indexSumarize.py [hunk 2]
```diff
@@ -99,21 +91,7 @@ def hundred_word_summary(decoded_summary):
...
-  # print(len(summary.split()))
-
-  # final_summary = hundred_word_summary(summary)
-  # print(final_summary)
-  # print(len(final_summary.split()))
+# if __name__== "__main__":
+#   raw_text = "The indian sapce is very gog jking to yo"
+#   summary = summarize(raw_text)
```
Comment (khareyash05) [inline server/src/indexSumarize.py:96]: what is the need of creating a new main and then commenting it ?

**Review by khareyash05 — COMMENTED** [review]
Kindly review

File: server/src/indexSumarize.py (hunk 1 above)
Comment (malayjoshi13) [reply server/src/indexSumarize.py:9]: Just merge it. I have tested it.

**Pull request merged**